pkgs/applications/editors/vim/configurable.nix

```diff
@@ -95,6 +94,7 @@ in stdenv.mkDerivation rec {
     "--disable-carbon_check"
     "--disable-gtktest"
   ]
+    ++ lib.optional (guiSupport == "gtk2" || guiSupport == "gtk3") "--enable-gui=${guiSupport}"
```

Comment:
Is that formatting right? It looks off to me.

Reply:
> Is that formatting right? It looks off to me.

Do you mean indentation? I checked that it evaluates with `guiSupport = false;`.

Reply:
> Do you mean indentation? 

yes